internal/controller/ols_app_server_reconciliator.go

```diff
@@ -25,13 +25,21 @@ func (r *OLSConfigReconciler) reconcileAppServer(ctx context.Context, olsconfig
 			Task: r.reconcileOLSConfigMap,
 		},
 		{
-			Name: "reconcile Deployment",
+			Name: "reconcile App Deployment",
 			Task: r.reconcileDeployment,
 		},
 		{
-			Name: "reconcile Service",
+			Name: "reconcile App Service",
 			Task: r.reconcileService,
 		},
+		{
+			Name: "reconcile Redis Deployment",
+			Task: r.reconcileRedisDeployment,
+		},
+		{
+			Name: "reconcile Redis Service",
+			Task: r.reconcileRedisService,
+		},
```

Comment:
we can have a dedicated file for reconciliating Redis resources such as `redis_reconciliator.go`

Reply:
Moved them to a separate file. Thank you.